src/features/metadata-instance-editor/Instance.js

```diff
@@ -569,6 +571,7 @@ class Instance extends React.PureComponent<Props, State> {
                         data-resin-target="metadata-instanceedit"
                         onClick={this.toggleIsEditing}
                         type="button"
+                        data-testid="metadata-instance-edit-button"
```

Comment:
RTL doesn't recommend using extra test attributes unless necessary, do we need to add a testid to this button? Are we able to find the IconEdit and click that instead?

Reply:
Now using the aria-label